code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -644,22 +647,109 @@
+	icon_state = "winterhood_mime"
+
+/obj/item/clothing/suit/hooded/wintercoat/clown
+	name = "\improper clown's winter coat"
+	desc = "A wintercoat made specially made for the clowns from an unknown place. All clowns have one, but not all of them wear it."
+	icon_state = "wintercoat_clown"
+	inhand_icon_state = "wintercoat_clown"
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/clown
+
+/obj/item/clothing/head/hooded/winterhood/clown
+	desc = "A bright yellow and red hood that screams honk just by looking at it, tends to temporarily blind moths with just a glance."
+	icon_state = "winterhood_clown"
+
+/obj/item/clothing/suit/hooded/wintercoat/chap
+	name = "\improper chaplins's winter coat"
+	desc = "Warm, Stylish, approved by the space pope. Everything you need in a winter coat."
+	icon_state = "wintercoat_chap"
+	inhand_icon_state = "wintercoat_chap"
+	allowed = list(/obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/healthanalyzer, /obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/pill, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/classic_baton/telescopic, /obj/item/soap, /obj/item/sensor_device, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman)
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/chap
+
+/obj/item/clothing/head/hooded/winterhood/chap
+	desc = "The Pope didnt forget to bless the most important part of the coat, thus giving off a holy feel."
+	icon_state = "winterhood_chap"
+
+/obj/item/clothing/suit/hooded/wintercoat/bartender
+	name = "\improper bartender's winter coat"
+	desc = "This is a winter coat made to look like a butler's suit."
+	icon_state = "wintercoat_bar"
+	inhand_icon_state = "wintercoat_bar"
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/toy, /obj/item/storage/fancy/cigarettes, /obj/item/lighter)
+	armor = list("melee" = 10, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/bar
+
+/obj/item/clothing/head/hooded/winterhood/bar
+	desc = "A hood tends to not be included with a butler's suit, but looks good on a bartender anyways."
+	icon_state = "winterhood_bar"
+
+/obj/item/clothing/suit/hooded/wintercoat/curator
+	name = "\improper curator's winter coat"
+	desc = "An acult looking wintercoat that the curator calls his own, he doesnt let anyone execpt himself touch it."
```

Comment:
"acult"?????

*doesn't

*except

this also assumes the curator's gender

Reply:
the description has been completely rewritten